rubygem/lib/zeus/load_tracking.rb

```
@@ -35,28 +35,17 @@ def features_loaded_by(&block)
       # then gets added to $LOADED_FEATURES array.
       def add_feature(file)
         full_path = File.expand_path(file)
+        return unless File.exist?(full_path)
```

this is such a weird api that silently ignores non-files...